src/components/Dropdown/Dropdown.driver.ts

```diff
@@ -0,0 +1,15 @@
+import {
+  BaseUniDriver,
+  baseUniDriverFactory,
+} from 'wix-ui-test-utils/base-driver';
+
+export interface DropdownDriver extends BaseUniDriver {
+  isMobile(): Promise<boolean>;
+}
+
+export const dropdownDriverFactory = (base: UniDriver): DropdownDriver => {
```

Comment:
Note that the `UniDriver` interface isn't imported (might be a problem in our generator).
It's just breaking the build.